src/lib.rs

```diff
@@ -136,20 +135,24 @@ impl manager::NotificationWatcher for ZWaveWatcher {
                 let controller = notification.get_controller();
                 let mut state = self.get_state();
                 if !state.controllers.contains(&controller) {
-                    println!("Found new controller: {:?}", controller);
+                    println!("Found new controller: {}", controller);
                     state.controllers.insert(controller);
                 }
             },
+            NotificationType::Type_NodeNew => {
+                let node = notification.get_node();
+                println!("NodeNew: {}", node);
```

Comment:
yeah, that's why I had a debug log for "_" :) I wouldn't mind uncommenting this but controlling using a program parameter (like -v :) )